test/lint/commit-script-check.sh

```diff
@@ -35,7 +35,7 @@ for commit in $(git rev-list --reverse $1); do
             git --no-pager diff --exit-code $commit && echo "OK" || (echo "Failed"; false) || RET=1
         fi
         git reset --quiet --hard HEAD
-     else
+     elif [ "$commit" != "f471a3be00c2b6433b8c258b716982c0539da13f" ]; then
```

Comment:
Looks good, but I'm curious why this doesn't hit upstream. (It's an upstream commit that has the issue, yeah?)

Reply:
Because it was merged into upstream before this check script was.

Reply:
(This is fine to ignore if you don't remember.)